Outkarm/blog-app
# add validation to project

**Description**
Adds validations for the User and Post models, as well as unit specs for all model methods and validations.

**Changes Made**
Added validates statements to the User and Post models for the required attributes and their constraints.
Wrote unit specs for all model methods and validations.
How to Test
Run rspec spec/models to run all model specs.
Run rubocop to ensure code style consistency.

**Checklist**
 

- [ ] Added validations to models.
- [ ]  Wrote unit specs for models.
- [ ]  Ran rspec spec/models to verify all tests passed.
- [ ]  Ran rubocop to ensure code style consistency.
- [ ]  Updated the README if necessary.

Let me know if you need any further changes!
